supervisor/snapshots/snapshot.py

```diff
@@ -481,3 +495,28 @@ def restore_repositories(self):
         Return a coroutine.
         """
         return self.sys_store.update_repositories(self.repositories)
+
+    def store_dockerconfig(self):
+        """Store the configuration for Docker."""
+        self.docker = {
+            ATTR_REGISTRIES: {
+                registry: {
+                    ATTR_USERNAME: credentials[ATTR_USERNAME],
```

Comment:
Should we also encrypt the username?

Reply:
In what scenario would that be needed?

Reply:
just a Suggestion

Reply:
So should this be encrypted as well? TBH I don't see why, but your call...

Reply:
I think it's fine to skip it for now, we can always add that later if needed